client/js/lounge.js

```diff
@@ -275,7 +286,7 @@ $(function() {
 
 			chat.find(".chan.active .chat").trigger("msg.sticky"); // fix growing
 		})
-		.tab(completeNicks, {hint: false})
+		.tab(completeNicks(false), {hint: false})
```

Comment:
What poor code about returning a function? That's like functional programming 101.